File: iniciante/python/1013.py
# -*- coding: utf-8 -*-

lin = input()
v = lin.split(" ")
a = int(v[0])
b = int(v[1])
c = int(v[2])
MaiorAB = (a+b+abs(a-b))/2
MaiorAB = (MaiorAB+c+abs(MaiorAB-c))/2
print("%d eh o maior" %MaiorAB)

File: iniciante/c/1013.c
#include <stdio.h>
#include <stdlib.h>
int main(){

	int A, B, C, MaiorAB;
	scanf("%d", &A);
	scanf("%d", &B);
	scanf("%d", &C);
	MaiorAB = (A + B + abs(A - B))/2;
	MaiorAB = (MaiorAB + C + abs(MaiorAB - C))/2;
	printf("%d eh o maior\n", MaiorAB);
}